primer.md

```diff
@@ -174,6 +175,15 @@ attributes may be defined that facilitate meeting persistence requirements but
 it is expected that these will continuously evolve along with industry best
 practices and advancements.
 
+The CloudEvents spec currently does not mandate or advocates any specific
+mechanism or principles in the matter of Authorization, Data Integrity and
+Confidentiality, as the core intention behind this spec is not to define
+Security principles with regards to CloudEvents. Every implementor has a
+different principle for enhancing their security model. We leave it up to the
```

Comment:
I appreciate that this speaks to the individual reading the primer.  It might be improved by also speaking to the community around that user.  The best choice for the reader is probably in the context of the community's offerings of concrete extensions and more abstract techniques so while the use of extensions seems like good guidance, a statement of the expectation that extensions will emerge might be a good nudge toward discovering what is on offer by the particular integrator or the community at large.

Reply:
Any suggestions on how can we improve the wording to address it to the community instead of the user?

Reply:
I like that it was addressing the individual.  I intended to suggest it was forgoing an opportunity to suggest a community opportunity and to direct users in looking to what the community will offer over time.  I have in mind a tiered offering of 1) attributes defined in the spec, 2) official extensions 3) unofficial but standardized extensions 4) individually defined/non-standard extensions.  You are saying we are at 4 but it might be worth suggesting 3 exists to be found via a search and that 2 should exist at some point (without making any promises).  I would expect this to encourage people to look which I would expect to be increasingly helpful over time but also provides incentive for offering such things and indicates there is an opportunity to impactfully do so.

Totally informally:
"hey, there should be extensions to find"
"maybe you'd like to make some for implementers to use"
"a business opportunity for hosting a turnkey and portable solution for these problem exists"

I suspect at the root of my comments is the durability of the text over time but mostly I'm trying to be helpful.  I made off comments and Doug skillfully used that to get me to comment.  Feel entirely welcome to disregard, I only mean to suggest possible improvements and if it doesn't feel like these changes would make it better, go with it as is.  :D